research/object_detection/dataset_tools/create_ava_tf_record.py

```diff
@@ -0,0 +1,439 @@
+          start_time = middle_frame_time - seconds_per_sequence // 2 - (
+              0 if seconds_per_sequence % 2 == 0 else 1)
+          end_time = middle_frame_time + (seconds_per_sequence // 2)
+
+          GLOBAL_SOURCE_ID += 1
+          total_xmins = []
+          total_xmaxs = []
+          total_ymins = []
+          total_ymaxs = []
+          total_labels = []
+          total_label_strings = []
+          total_images = []
+          total_source_ids = []
+          total_confidences = []
+          total_is_annotated = []
+          windowed_timestamp = start_time
+          while windowed_timestamp < end_time:
+            if (media_id, windowed_timestamp) in frame_excluded:
+              end_time += 1
+              windowed_timestamp += 1
+              logging.info("Ignoring and skipping excluded frame.")
+              continue
+
+            cur_vid.set(cv2.CAP_PROP_POS_MSEC,
+                        (windowed_timestamp) * SECONDS_TO_MILLI)
+            _, image = cur_vid.read()
+            _, buffer = cv2.imencode('.jpg', image)
+
+            bufstring = buffer.tostring()
+            total_images.append(dataset_util.bytes_feature(bufstring))
+            source_id = str(GLOBAL_SOURCE_ID) + "_" + media_id
+            total_source_ids.append(dataset_util.bytes_feature(
+                source_id.encode("utf8")))
+            total_is_annotated.append(dataset_util.int64_feature(1))
+            GLOBAL_SOURCE_ID += 1
+
+            xmins = []
+            xmaxs = []
+            ymins = []
+            ymaxs = []
+            labels = []
+            label_strings = []
+            confidences = []
+            for row in frame_annotations[(media_id, windowed_timestamp)]:
+              if len(row) > 2 and int(row["action_label"]) in label_map:
+                xmins.append(float(row["xmin"]))
+                xmaxs.append(float(row["xmax"]))
+                ymins.append(float(row["ymin"]))
+                ymaxs.append(float(row["ymax"]))
+                labels.append(int(row["action_label"]))
+                label_strings.append(label_map[int(row["action_label"])])
+                confidences.append(1)
+              else:
+                logging.warning("Unknown label: %s", row["action_label"])
+
+            #Display the image and bounding boxes being
+            #processed (for debugging purposes) if desired.
+
+            total_xmins.append(dataset_util.float_list_feature(xmins))
+            total_xmaxs.append(dataset_util.float_list_feature(xmaxs))
+            total_ymins.append(dataset_util.float_list_feature(ymins))
+            total_ymaxs.append(dataset_util.float_list_feature(ymaxs))
+            total_labels.append(dataset_util.int64_list_feature(labels))
+            total_label_strings.append(
+                dataset_util.bytes_list_feature(label_strings))
+            total_confidences.append(
+                dataset_util.float_list_feature(confidences))
+            windowed_timestamp += 1
+
```

Comment:
please reuse https://github.com/tensorflow/models/blob/master/research/object_detection/dataset_tools/seq_example_util.py#L149 to construct sequence example.